core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -599,6 +600,13 @@ public void start(Boolean isPrimary) throws IOException {
               .format("Unauthorized user on root. inode owner: %s current user: %s",
                   root.getOwner(), serverOwner)));
         }
```

Comment:
The `MountTableTrie` is initialized and started when `DefaultFileSystemMaster.start` is called